.gitignore

```diff
@@ -25,4 +25,7 @@ htdocs/js/components/.module-cache/
 node_modules/
 modules/*/js/*.map
 htdocs/js/components/*.map
-npm-debug.log*
\ No newline at end of file
+npm-debug.log*
+SQL/Archive/autogenerated/reversion/*.sql
+SQL/Archive/autogenerated/single_use/*.sql
+SQL/Archive/autogenerated/*.sql
```

Comment:
does this need to be here if patches are not supposed to be added directly into this directory anyways? maybe it will be a good way to catch this mistake if we have it tracked by github

Reply:
https://git-scm.com/docs/gitignore#_notes

Reply:
^ please elaborate

Reply:
https://www.atlassian.com/git/tutorials/saving-changes/gitignore